ProcessLib/SmallDeformationWithLIE/LocalAssembler/CreateLocalAssemblers.h

```diff
@@ -0,0 +1,113 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef PROCESSLIB_SMALLDEFORMATION_WITH_LIE_CREATE_LOCAL_ASSEMBLERS_H_
+#define PROCESSLIB_SMALLDEFORMATION_WITH_LIE_CREATE_LOCAL_ASSEMBLERS_H_
+
+#include <vector>
+
+#include <logog/include/logog.hpp>
+
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+
+#include "LocalDataInitializer.h"
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+namespace detail
+{
+template <unsigned GlobalDim, int DisplacementDim,
+          template <typename, typename, unsigned, int>
+          class LocalAssemblerMatrixImplementation,
+          template <typename, typename, unsigned, int>
+          class LocalAssemblerMatrixNearFractureImplementation,
+          template <typename, typename, unsigned, int>
+          class LocalAssemblerFractureImplementation,
+          typename LocalAssemblerInterface, typename... ExtraCtorArgs>
+void createLocalAssemblers(
+    NumLib::LocalToGlobalIndexMap const& dof_table,
+    std::vector<MeshLib::Element*> const& mesh_elements,
+    std::vector<std::unique_ptr<LocalAssemblerInterface>>& local_assemblers,
+    ExtraCtorArgs&&... extra_ctor_args)
+{
+    // Shape matrices initializer
```

Comment:
"matrices"  is correct, because it includes multiple vector and matrices, i.e. N, dN/dr, dN/dx